### ISSUE TITLE
Can't start session for trinity

### ISSUE THREAD

**Issue by wdm78** [alces-software/flight-aws-marketplace#40]
Can't start a session for trinity - tested on flight 2016.2:

```
[alces@login1(scooby) ~]$ alces session enable trinity
  >>> Loaded plugins: fastestmirror
  >>> Cannot open: http://mirror.ppa.trinitydesktop.org/trinity/trinity/rpm/el7/trinity-r14/RPMS/noarch/trinity-repo-14.0.2-1.el7.noarch.rpm. Skipping.
  >>> Error: Nothing to do
  >>> Sorry, the Trinity repository is not available.
alces session enable: unable to enable session: base/trinity
```

**Comment by mjtko** [comment]
Refs alces-software/clusterware-sessions#8.

**Comment by mjtko** [comment]
This was resolved in Clusterware 1.5.1/2016.2r2.
